nostr-client-plus/src/bin/aggregator.rs

```diff
@@ -270,3 +347,46 @@ fn register_metrics() {
         .register(Box::new(FINISHED_JOBS.clone()))
         .expect("Cannot register finished_jobs");
 }
+
+fn validate_result_event(event: EventOnWire, assign_event: EventOnWire) -> Result<bool> {
+    let payload: ResultPayload = serde_json::from_str(event.content.as_str())?;
+    let job_type = payload.header.job_type;
+    // POW event check
+    if job_type == 0 {
+        // Check assignee is Result Sender
+        if let Ok(Some(assignee)) = get_single_tag_entry('p', &assign_event.tags) {
```

Comment:
with data classification, we will have multiple assignees for one job so here we should not use get_single_tag_entry but get a list of users and check if the assignee is included in the list

Reply:
Here I check job_type = 0 , it's for pow task only